akka-javasdk/src/main/scala/akka/javasdk/impl/reflection/Reflect.scala

```diff
@@ -160,6 +160,27 @@ private[impl] object Reflect {
     loop(workflow.getClass).asInstanceOf[Class[S]]
   }
 
+  def tableUpdaterRowType(tableUpdater: Class[_]): Class[_] = {
+    @tailrec
+    def loop(current: Class[_]): Class[_] =
+      if (current == classOf[AnyRef])
+        // recursed to root without finding type param
+        throw new IllegalArgumentException(s"Cannot find table updater class for ${tableUpdater.getClass}")
+      else {
+        current.getGenericSuperclass match {
+          case parameterizedType: ParameterizedType =>
+            if (parameterizedType.getActualTypeArguments.size == 1)
+              parameterizedType.getActualTypeArguments.head.asInstanceOf[Class[_]]
+            else throw new IllegalArgumentException(s"Cannot find table updater class for ${tableUpdater.getClass}")
+          case noTypeParamsParent: Class[_] =>
+            // recurse and look at parent
+            loop(noTypeParamsParent)
+        }
+      }
+
+    loop(tableUpdater)
+  }
```

Comment:
Use this new method here as well, for consistency: https://github.com/akka/akka-sdk/blob/eda731043bc811431ba9f123a4c1c206731b4d45/akka-javasdk/src/main/scala/akka/javasdk/impl/view/ViewDescriptorFactory.scala#L68

Reply:
done